src/AWS.Deploy.ServerMode.Client/ServerModeHttpClient.cs

```diff
@@ -61,7 +61,7 @@ public static void AddAuthorizationHeader(HttpRequestMessage request, ImmutableC
                 {"awsAccessKeyId", credentials.AccessKey },
                 {"awsSecretKey", credentials.SecretKey },
                 {"requestId", Guid.NewGuid().ToString() },
-                {"issueDate", DateTime.UtcNow.ToString("yyyy-MM-dd'T'HH:mm:ss.fffzzz", DateTimeFormatInfo.InvariantInfo) }
```

Comment:
I never liked these formats, can we use simple good epoch time? 

https://www.epochconverter.com/

Reply:
Most APIs usually use the a more human readable form like the one we use. I am not a fan of epoch time.

Reply:
I can disagree and commit here.

Just like we never store the age, we save date of birth. Time should be agnostic of all factor.

Epoch time 
- doesn't have timezone
- one universal format works on all platforms
- no issues with daylight saving time
- apps can convert it local time based on their needs

Yes it comes with readability issue but there are tones of tools to convert it.

Reply:
The format we are using "ISO 8601" should have the benefits of epoch time. So we should be fine.

Reply:
Epoch dates are rarely used in pure .NET solutions.